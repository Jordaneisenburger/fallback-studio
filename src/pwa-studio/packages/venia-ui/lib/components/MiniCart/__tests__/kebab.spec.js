import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Kebab from '../kebab';

const renderer = new ShallowRenderer();

test('it renders correctly without children', () => {
    const tree = renderer.render(<Kebab />);

    expect(tree).toMatchSnapshot();
});

test('it renders children passed to it', () => {
    const tree = renderer.render(
        <Kebab>
            <div>Child 1</div>
            <div>Child 2</div>
            <div>Child 3</div>
        </Kebab>
    );

    expect(tree).toMatchSnapshot();
});
