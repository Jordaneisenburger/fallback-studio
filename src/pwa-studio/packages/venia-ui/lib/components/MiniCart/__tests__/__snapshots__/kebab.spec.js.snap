// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders children passed to it 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    <Classify(Icon)
      src={[Function]}
    />
  </button>
  <ul>
    <div>
      Child 1
    </div>
    <div>
      Child 2
    </div>
    <div>
      Child 3
    </div>
  </ul>
</div>
`;

exports[`it renders correctly without children 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    <Classify(Icon)
      src={[Function]}
    />
  </button>
  <ul />
</div>
`;
